baremetal/metal3machine_manager.go

```diff
@@ -105,7 +106,7 @@ func NewMachineManager(client client.Client,
 // SetFinalizer sets finalizer
 func (m *MachineManager) SetFinalizer() {
 	// If the Metal3Machine doesn't have finalizer, add it.
-	if !util.Contains(m.Metal3Machine.Finalizers, capm3.MachineFinalizer) {
+	if !Contains(m.Metal3Machine.Finalizers, capm3.MachineFinalizer) {
```

Comment:
What is the motivation behind not using the util functions and writing own?

Reply:
Cluster API removed the Contains and Filter functions from their util package in the latest version